### ISSUE TITLE
Streamlit-folium is extending its container way further past the bottom border of the map

### ISSUE THREAD

**Issue by espogian** [randyzwitch/streamlit-folium#213]
The current version of streamlit-folium (0.22.1) has introduced a bug which is clearly visible from the showcase page (https://folium.streamlit.app/). Right on the bottom of the map, there is a huge blank space which extends a lot until the next element or the footer/bottom of the page.

**Comment by espogian** [comment]
I can see the exact same problem (the issue of the blank space after the map, which I described in my first post) present on the published version of https://folium.streamlit.app/ in my apps which are using streamlit-folium (0.22.1). However, the error you are posting is not present in my apps, therefore I think is not related. Thank you

**Comment by randyzwitch** [comment]
The issue you are seeing isn't present for me? 

<img width="1578" alt="Screenshot 2024-09-17 at 9 55 32 AM" src="https://github.com/user-attachments/assets/f89736f7-da7d-4881-9a2e-8a55e9077bd7">

**Comment by espogian** [comment]
I can scroll way past the end of the map. Reproduced using Chrome Version 128.0.6613.138 (Official Build) (arm64) and Safari Version 18.0 (19619.1.26.111.10, 19619).
With both of them I've tried to access the https://folium.streamlit.app/ clearing the cache.

![image](https://github.com/user-attachments/assets/39d4c5d5-e380-47a3-bb06-9d642a1bb1c4)

**Comment by randyzwitch** [comment]
I can get that to happen, but it's intermittent. I've rebooted the app to ensure its on 0.22.1, and will fix the experimental_rerun error to see if this persists. If so, then we can try and figure out what else might be causing this

**Comment by randyzwitch** [comment]
What is you want from me? I'm trying to figure out the issue and I'm telling you that on my computer it is intermittent. That's not up for debate, it's what I am seeing.

**Comment by randyzwitch** [comment]
Thanks for clarifying :)

I now agree that it's not intermittent, it's literally every other refresh for me, irrespective of having/not having cache. Unfortunately, there are no messages in the console to indicate an error.

@hansthen I don't _think_ this is an issue with the patch you just did for Draw, but I also don't know what to look for here. Any ideas?

**Comment by randyzwitch** [comment]
As it turns out, it's not the map but the code on the right side that is causing the issue. The map is only 700px tall, the right column is 2438px tall

<img width="1625" alt="Screenshot 2024-09-17 at 10 42 42 AM" src="https://github.com/user-attachments/assets/7e1483c2-7f3e-49b4-8524-3524ea8d39c0">
<img width="797" alt="Screenshot 2024-09-17 at 10 42 30 AM" src="https://github.com/user-attachments/assets/ec6ddedd-def2-440d-b9f0-7afe5af9ff8e">


When it doesn't show the issue, the code column is only 833px tall

<img width="2038" alt="Screenshot 2024-09-17 at 10 44 55 AM" src="https://github.com/user-attachments/assets/1b4293c1-d36e-4a28-a435-7f58cc169528">

**Comment by hansthen** [comment]
> @hansthen I don't _think_ this is an issue with the patch you just did for Draw, but I also don't know what to look for here. Any ideas?

Just to be sure, I undid my changes for the Draw plugin on the latest main branch locally. Without my changes the demo app also shows this issue. 

The same issue also shows up in the page for the draw plugin. There is a code block below the two columns, which makes it easier to see. The issue is still there even if there is nothing in the second column.
```
  with st.echo(code_location="below"):
      import folium
      import streamlit as st
      from folium.plugins import Draw
  
      from streamlit_folium import st_folium
  
      m = folium.Map(location=[39.949610, -75.150282], zoom_start=5)
      Draw(export=True).add_to(m)
  
      c1, c2 = st.columns(2)
      with c1:
          output = st_folium(m, width=700, height=500)
  
      with c2:
          pass
```

**Comment by hansthen** [comment]
I don't have time right now, but I would check the `setComponentHeight` message sent from the `streamlit-folium` component in a javascript debugger.

**Comment by randyzwitch** [comment]
Fixed by #237
